tfhe/src/shortint/server_key/mod.rs

```diff
@@ -2,40 +2,42 @@
 //!
 //! This module implements the generation of the server public key, together with all the
 //! available homomorphic integer operations.
-mod add;
-mod bitwise_op;
-mod comp_op;
-mod div_mod;
-mod mul;
-mod neg;
-mod scalar_add;
-mod scalar_bitwise_op;
-mod scalar_mul;
-mod scalar_sub;
-mod shift;
-mod sub;
+pub(crate) mod add;
```

Comment:
still unclear what requires this change, though it's not an issue per se

Reply:
I guess it was required while I was doing changes
It's currently not
I'll remove it

Reply:
thanks, I think there is something similar for the wop somewhere else